cfgov/unprocessed/css/molecules/nav-link.less

```diff
@@ -0,0 +1,80 @@
+/* =========================================================================
+   cfgov-refresh
+   m-nav-link
+   ========================================================================= */
+
+/* topdoc
+  name: Nav Link
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <a class="m-nav-link"
+           href="#">
+        </a>
+      codenotes:
+        - ".m-nav-link"
+    - name: Parent link modifier
+      markup: |
+        <a class="m-nav-link
+                  m-nav-link__parent"
+           href="#">
+           Parent nav link
+        </a>
+      codenotes:
+        - ".m-nav-link.m-nav-link__parent"
+      notes:
+        - "Parent links are styled as heading-4."
+    - name: Current link modifier
+      markup: |
+        <a class="m-nav-link
+                  m-nav-link__current"
+           href="#">
+           Current nav link
+        </a>
+      codenotes:
+        - ".m-nav-link.m-nav-link__current"
+      notes:
+        - "Current links have a left green border on all screens."
+  tags:
+    - cfgov-molecules
+*/
+
+.m-nav-link {
+    .u-inline-block();
+
+    .u-link__colors(@pacific, @pacific, @black, @black, @black,
+                    transparent, transparent, @green, @green, @green);
+
+    border-left-style: solid;
+    border-left-width: 5px;
+
+    .respond-to-max(@bp-sm-max, {
+        display: block;
+
+        padding: unit((@grid_gutter-width / 2) / @base-font-size-px, em);
+
+        .u-link__colors(@darkgray, @darkgray, @darkgray, @darkgray,
+                        @darkgray, transparent, transparent, @green,
+                        @green, @green);
+    });
+
+    .respond-to-min(@bp-med-min, {
+        padding-top: unit(10px / @base-font-size-px, em);
+        padding-bottom: unit(10px / @base-font-size-px, em);
+        padding-left: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
+    });
+
+    &__current {
+        .u-link__colors(@black, @black, @black, @black, @black,
+                        @green, @green, @green, @green, @green);
+
+        cursor: default;
+    }
+
+    &__parent {
+        .heading-4();
+
+        margin-bottom: inherit;
```

Comment:
It will always inherit from the `li` it lives within (which currently have a default of `.5em`). If the nav items ever end up having a non-default margin, this makes it automatically inherit that rather than changing it in two places.